brave/browser-laptop
# Improved errors when loading ledger balance

Humanizes errors when loading ledger balance by mapping error codes to localized messages.

Fix #4920

Auditors: @luixxiul

Test Plan:
1. Disable internet for Brave (can disable system wide, or app specific eg w/ Little Snitch).
2. Open Brave and view Preferences > Payments.
3. Account balance should show error and details.
